app/models/vacancy.rb

```diff
@@ -252,6 +252,14 @@ def skip_update_callbacks?
     @skip_update_callbacks.present?
   end
 
+  def show_supporting_documents?
+    UploadDocumentsFeature.enabled? && !show_candidate_specification?
+  end
+
+  def show_candidate_specification?
+    experience.present? && qualifications.present? && education.present?
```

Comment:
Is it possible for a pre-upload-docs vacancy to only have a subset of these attributes present?
Are these required fields in the candidate specification form?

Reply:
The answers are respectively 'no' and 'yes' so this is resolved.

Reply:
They are required fields. This method would change when the behaviour for changing candidate specification in an upload documents world is defined. Will add a TODO in the code!